src/components/views/dialogs/ModuleUiDialog.tsx

```diff
@@ -17,15 +17,18 @@ limitations under the License.
 import React, { createRef } from "react";
 import { DialogContent, DialogProps } from "@matrix-org/react-sdk-module-api/lib/components/DialogContent";
 import { logger } from "matrix-js-sdk/src/logger";
+import { ModuleApi } from "@matrix-org/react-sdk-module-api/lib/ModuleApi";
+import { ModuleUiDialogOptions } from "@matrix-org/react-sdk-module-api/lib/types/ModuleUiDialogOptions";
 
 import ScrollableBaseModal, { IScrollableBaseState } from "./ScrollableBaseModal";
 import { _t } from "../../../languageHandler";
 
 interface IProps<P extends DialogProps, C extends DialogContent<P>> {
     contentFactory: (props: P, ref: React.RefObject<C>) => React.ReactNode;
-    contentProps: P;
-    title: string;
-    onFinished(ok?: boolean, model?: Awaited<ReturnType<DialogContent<P>["trySubmit"]>>): void;
+    contentProps: Omit<P, keyof DialogProps> | undefined;
```

Comment:
wonder if this should be called `additionalContentProps` to distinguish it from the actual `contentProps`